src/main/scala/pro/civitaspo/digdag/plugin/ecs_task/wait/EcsTaskWaitOperator.scala

```diff
@@ -17,14 +19,28 @@ class EcsTaskWaitOperator(operatorName: String, context: OperatorContext, system
   val status: String = params.get("status", classOf[String], "STOPPED")
   val ignoreFailure: Boolean = params.get("ignore_failure", classOf[Boolean], false)
   val ignoreExitCode: Boolean = params.get("ignore_exit_code", classOf[Boolean], false)
+  val delayStrategyStr: String = params.get("delay_strategy", classOf[String], "fixed")
```

Comment:
In this plugin, we try to match the variable created by parsing option to the name of option. So please change to `val delayStrategy`.

Reply:
Nice policy. I follow your instruction.

Reply:
I fixed as `polling_strategy`. thanks!